geometry_msgs/PoseStamped target
bool completed
---
bool running
